src/rules/selector-no-union-class-name/index.js

```diff
@@ -0,0 +1,51 @@
+import { utils } from "stylelint";
+import { namespace, parseSelector } from "../../utils";
+import { isClassName, isCombinator } from "postcss-selector-parser";
+
+export const ruleName = namespace("selector-no-union-class-name");
+
+export const messages = utils.ruleMessages(ruleName, {
+  rejected: "Union class name detected"
```

Comment:
The format for stylelint error messages is "Expected [something] [in some context]" and "Unexpected [something] [in some context]". Could you please change this one to match that format?

https://github.com/stylelint/stylelint/blob/master/docs/developer-guide/rules.md#determine-violation-messages

Reply:
How about `Unexpected class name in parent of '&'` ?

Reply:
It could match the description, e.g. `Unexpected union class name with the parent selector (&)`.

Reply:
Fixed: a83dc5f